unit_test/extensions/onnx/mnist_data/data_generator.py

```diff
@@ -0,0 +1,65 @@
+#%%
+
+from mnist import MNIST;
+import base64;
+import numpy as np;
+import struct;
+
+
+#%%
```

Comment:
What are these for?

Reply:
These represent cell boundaries for the built-in "notebook-like" experience for Python Tools in VS Code